src/textual/screen.py

```diff
@@ -621,6 +674,41 @@ def focus_previous(self, selector: str | type[QueryType] = "*") -> Widget | None
         """
         return self._move_focus(-1, selector)
 
+    def maximize(self, widget: Widget, container: bool = True) -> None:
+        """Maximize a widget, so it fills the screen.
+
+        Args:
+            widget: Widget to maximize.
+            container: Maximize container if possible.
```

Comment:
Could you add a bit more detail to the `container` part of this docstring?

Reply:
k